packages/hardhat/test/e2e/crab-Migration.ts

```diff
@@ -69,105 +69,108 @@ describe("Crab Migration", function () {
         await provider.send('evm_mine', []);
         await provider.send('hardhat_stopImpersonatingAccount', [crabV1Whale2]);
 
-    })    
-    
+    })
+
     this.beforeAll("Deploy Crab 2", async () => {
         const CrabContract = await ethers.getContractFactory("CrabStrategyV2");
         crabStrategyV2 = (await CrabContract.deploy(
-                            squeethControllerAddress, 
-                            oracleAddress, 
-                            wethAddress, 
-                            uniswapFactoryAddress, 
-                            wethOsqthPoolAddress, 
-                            random.address,
-                            1,
-                            1));
-        
+            squeethControllerAddress,
+            oracleAddress,
+            wethAddress,
+            uniswapFactoryAddress,
+            wethOsqthPoolAddress,
+            random.address,
+            1,
+            1,
+            1,
+            1,
```

Comment:
why 4 of these? Crab v2 the latest version should only have 2.

Reply:
does the test run when you change these?

Reply:
It failed for me in local. I am fixing it